eq-publisher/src/eq_schema/Answer.js

```diff
@@ -155,14 +153,16 @@ class Answer {
     if (description) {
       option.description = description;
     }
+    if (additionalAnswer) {
+      option.detail_answer = {
```

Comment:
is there a reason for using snake case?

Reply:
Yeah Runner expects snake case instead of camel unfortunately.